content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -98,7 +98,7 @@
 
 	@Setter
 	private ContentHostingService contentHostingService;
-	
+
```

Comment:
remove these whitespace changes

Reply:
Not sure how I can, I don't know why those are there

Reply:
There are many methods to do this. First you'll need to reset HEAD~1 so that you can edit your commit. Then use a "git difftool" to be able to step through each change and decide whether to undo it or not